podman/api_connection.py

```diff
@@ -13,20 +13,26 @@
 
 
 class ApiConnection(HTTPConnection, AbstractContextManager):
-    """ ApiConnection provides a specialized HTTPConnection
-        to a Podman service."""
-
-    def __init__(self, url, base="/v1.24/libpod", *args,
-                 **kwargs):  # pylint: disable-msg=W1113
+    """
```

Comment:
I've been using rst example doc strings. I guess we should establish a standard. I wouldn't add the line break here and leave it as it.  could remove the leading space to be consistent with everything else.

Reply:
Got it. I can fix this ASAP and adjust to the rst example docstrings, no probs.